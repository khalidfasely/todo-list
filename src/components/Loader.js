const Loader = () => (
    <div className="loader-page">
        <div className="loader-container">
            <div className="loader">
                <h2 data-text='Loading...'>Loading...</h2>
            </div>    
        </div>
    </div>
);

export default Loader;