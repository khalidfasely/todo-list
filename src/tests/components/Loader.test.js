import React from 'react';
import { render } from '@testing-library/react';
import Loader from '../../components/Loader';

test('Should render Loader component correctly', () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
});