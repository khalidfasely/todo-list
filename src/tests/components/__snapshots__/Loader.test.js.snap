// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Loader component correctly 1`] = `
<DocumentFragment>
  <div
    class="loader-page"
  >
    <div
      class="loader-container"
    >
      <div
        class="loader"
      >
        <h2
          data-text="Loading..."
        >
          Loading...
        </h2>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
